### ISSUE TITLE
Incompatibility with freetype 2.10+

### ISSUE THREAD

**Issue by djhoese** [rougier/freetype-py#107]
Issue: vispy's tests are now failing with missing symbols from freetype:
https://travis-ci.org/vispy/vispy/jobs/507194615#L3365

Error:

```
  File "/home/travis/build/vispy/vispy/vispy/ext/freetype.py", line 12, in <module>
    from freetype import *  # noqa
  File "/home/travis/anaconda/envs/testenv/lib/python3.7/site-packages/freetype/__init__.py", line 21, in <module>
    from freetype.raw import *
  File "/home/travis/anaconda/envs/testenv/lib/python3.7/site-packages/freetype/raw.py", line 249, in <module>
    FT_Outline_Done_Internal       = _lib.FT_Outline_Done_Internal
  File "/home/travis/anaconda/envs/testenv/lib/python3.7/ctypes/__init__.py", line 369, in __getattr__
    func = self.__getitem__(name)
  File "/home/travis/anaconda/envs/testenv/lib/python3.7/ctypes/__init__.py", line 374, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))
AttributeError: /home/travis/anaconda/envs/testenv/lib/./libfreetype.so.6: undefined symbol: FT_Outline_Done_Internal
```

I'm a little worried by the message on https://www.freetype.org/

> Additionally, the API reference has been completely overhauled and modernized, as already announced.

See the linked travis job for more environment information.

By limiting the vispy tests to freetype<2.10 the tests pass.

**Comment by HinTak** [comment]
I will have a look in the next few days, after I upgrade.

**Comment by madig** [comment]
You already fixed this in https://github.com/rougier/freetype-py/commit/4b03afebfa9ecd5ea8badf64e7621640ae7f068c. We just need to release a new version :) I'll have a look.

**Comment by madig** [comment]
Fixed in 2.1.0.

**Comment by HinTak** [comment]
:-)
